### ISSUE TITLE
Create aliyun server's environment

### ISSUE THREAD

**Issue by wangqing1218** [ACLoong/Meeting-Review#4]
1. Install **java8** into it.
2. Install **python2** and python3 into it.
3. into **maven3** into it.
4. into **cmake** 3 or later into it.
5. into **git** for it.

**Comment by ghost** [comment]
I want install samba server into  the aliyun VPS.
Could I?Or mayber I should create another issue.

**Comment by wangqing1218** [comment]
This issue is ok. 
And I dont know about samba.
I think u can install it by urself.

**Comment by ghost** [comment]
Installation failed, I need root permissoon, or will @MrKingHH to install the environment?

**Comment by MrKingHH** [comment]
sorry, i haven't received message in my email box.

**Comment by ghost** [comment]
Git:I have install git ,but didn't set the git conf.And if anyone want to try it could get the ref [here](https://www.aliyun.com/jiaocheng/133619.html),Besides I don't konw why we should install git into the server.Because we already have the github repo and it seems safely...
Cmake:the network is too slow so that wget cannot work so fast, I will try next day.
### python2 & python3 has been installed
### Java8:install success(javac -version   javac 1.8.0_171)

**Comment by ghost** [comment]
### install cmake finished

wuwendong@ACLoong:~/env$ cmake --version
cmake version 3.5.1

**Comment by ghost** [comment]
### ADD: install google test framework

usage:
#include <gtest/gtest.h>  in cpp file

then, 
g++ test.cpp -lgtest_main -lgtest -lpthread -o test
or 
g++ test.cpp  /usr/local/lib/libgtest.a -lpthread -o test

**Comment by wangqing1218** [comment]
Well done.

**Comment by ghost** [comment]
append `alias python="python3"`  to ~/.profile while you want to use python3 instead of python2.7

**Comment by jovany-wang** [comment]
@throcean  You mean that `which python` will output `python3.x.x`?

**Comment by ghost** [comment]
@wangqing1218 No,the shell instruction `alias python="python3"` will playing a part when you want to enter the Python3 interactive window.In general we use `python` to interactive with python2, now we could interactive with python3 by `python` after `alias` worked.
And the `alias` just specified the role when using `python`,it‘s useless while you want to use `which` or `whereis` to find the python executable file.

**Comment by ghost** [comment]
### ADD:Docker, version 18.09.0

**Comment by jovany-wang** [comment]
@throcean  What the python version of that machine? 
And I think we should use virtual-env to instead of native python env.

**Comment by ghost** [comment]
@jovan-wong There has python2.7 and python3.5 in machine. I do agree with you.There are many choice like virtual-env,conda, etc.

**Comment by jovany-wang** [comment]
@throcean  Let install anaconda as well.
And enable it when we login the server.
I think we should use python 3 in anaconda by default.

**Comment by ghost** [comment]
### ADD:Anaconda
```
conda --version
conda 4.5.11
```
I will write a short usage about conda later. Besides, I have removed `python2.7` and `python3.7` which installed by `apt-get`
